pymongo/helpers.py

```diff
@@ -103,14 +103,16 @@ def _index_document(index_list):
 
 
 def _check_command_response(response, msg=None, allowable_errors=None,
-                            parse_write_concern_error=False):
+                            parse_write_concern_error=False,
+                            max_wire_version=None):
```

Comment:
I suggest we make `max_wire_version` a required argument to ensure we don't forget to pass it.

Reply:
Done. Note that I used None as the `max_wire_version` value in several tests that directly use this method.